### ISSUE TITLE
$75 | Accessibility Issue:  Cannot Access Menu Items Using Keyboard

### ISSUE THREAD

**Issue by VanitaW** [topcoder-platform/navigation-component#132]
***Reproduction Steps:***
1. Go to https://test-community-app.topcoder-dev.com/challenges 
2. Sign in with username: dan_developer  password: dantopcoder123
3. Click on 'log in'
4.  Navigate the site using only the keyboard.
5. Observe that  the navigation bar is inaccessible using the keyboard

***Expected Behavior***
To be able to navigate the menu using only the keyboard

***Actual Behavior***
Nothing happens when the keyboard is pressed

***Device/OS Information:***
macOS
Safari
12.1.1

**Comment by luizrrodrigues** [comment]
@LieutenantRoger Yes looks right, but need able to access menu links via ENTER key, in your solution I'm able to navigate but if press ENTER nothing happen. (So can't access sub menu too)

**Comment by LieutenantRoger** [comment]
@luizrrodrigues  more fix is pushed here: 
https://github.com/topcoder-platform/navigation-component/pull/122

**Comment by luizrrodrigues** [comment]
@LieutenantRoger Now I can Press Enter , but still not able to access sub menu.

**Comment by LieutenantRoger** [comment]
Hi @luizrrodrigues  have you seen what I'm seeing ?

https://monosnap.com/file/UbBly2NmQs2ZO6ejZUZDzj0jFJEzwz

**Comment by luizrrodrigues** [comment]
@LieutenantRoger Let me re check this, I remember until search works same way here, but not good to sub menu. I'll recheck and let you know.

**Comment by LieutenantRoger** [comment]
Hi, how is this going ?

**Comment by LieutenantRoger** [comment]
@luizrrodrigues  how is it going for this ticket ?

**Comment by luizrrodrigues** [comment]
Hey @LieutenantRoger sorry delay here. Was working in other high priority items.
I'll check these remain issues this weekend and make sure to members paid.

**Comment by luizrrodrigues** [comment]
@LieutenantRoger Hey, finally I'm getting back to this one.

Please check the video to see the issue:
https://drive.google.com/file/d/1LXgrtnqPbAYRH740Snef3BJGH5nr0sHh/view?usp=sharing

In your video looks like you are able to move to submenu after the search menu, but here in Safari (Mac) after leave search menu go to the main page.

Also, the flow shared in your video not right, we need to go to sub-menu after press ENTER, like bellow:
1 - Tab until the "Tracks" menu
2 - Press ENTER
3 - Press Tab need navigate between sub-menu of "Tracks"

Make sense? Is possible?

Please let me know, thanks.
